decidim-blogs/app/views/decidim/blogs/admin/posts/manage_trash.html.erb

```diff
@@ -0,0 +1,19 @@
+<% add_decidim_page_title(t(".title")) %>
+<div class="card">
+  <div class="item_show__header">
+    <h1 class="item_show__header-title">
+      <%= t(".title") %>
+    </h1>
+  </div>
+  <div class="table-scroll">
+    <table class="table-list">
+      <%= render partial: "posts-thead" %>
+      <tbody>
+      <% trashable_deleted_collection.created_at_desc.each do |post| %>
```

Comment:
just asking, maybe we should sort desc by deletion date, so that most recent deleted element to first in list?